run

```diff
@@ -660,9 +684,9 @@ done
 
 for d in "${DISKS[@]}"; do
     IFS=':' read -ra disk <<< "$d"
-    disk_path="$USR_CACHE_DIR/$VMNAME/disk-${disk[0]}.qcow2"
-    log qemu-img create -f qcow2 "$disk_path" ${disk[2]}
-    VOLUMES+=("disk-${disk[0]}-$OWNER:$disk_path:qcow2:${disk[1]}:${disk[0]}")
+    disk_path="$USR_CACHE_DIR/$VMNAME/disk-${disk[0]}.raw"
```

Comment:
Why you remove qcow2?

Reply:
You can never use a qcow2 image for a shared-write disk as it will cause corruption of the qcow2 metadata. Use raw format for shared disks.